app/data/2_0001.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Fair enough if it's already agreed across systems. Let's push for some meaningful names going forward this means we don't need mappings and it's readable